spec/faraday/options/env_spec.rb

```diff
@@ -27,6 +27,12 @@
     expect(ssl.fetch(:verify, true)).to be_falsey
   end
 
+  it 'handle verify_hostname when fetching' do
+    ssl = Faraday::SSLOptions.new
+    ssl.verify_hostname = true
+    expect(ssl.fetch(:verify_hostname, true)).to be_truthy
```

Comment:
Sorry maybe I wasn't entirely clear in my previous comment.
I can see you fixed the property name, which is great, but it's not a good idea to test `be_truthy` when you default the `fetch` to `true`, as this might cause false-positives.

If you check the example above (for `ssl.verify`), the value is set to false and the condition is `be_falsey`. You could do that here as well, or if you prefer to keep the `true` value then please change the `fetch` to default to `false`.

Reply:
Oops. I got it.